packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -122,17 +122,19 @@ interface EntryValidationTextProps {
 
 const EntryValidationText = ({ action, schema, entry }: EntryValidationTextProps) => {
   const { formatMessage } = useIntl();
-  const { validate } = unstable_useDocument(
+
+  const { validate, isLoading } = unstable_useDocument(
     {
       collectionType: schema?.kind ?? '',
       model: schema?.uid ?? '',
     },
     {
-      skip: !schema,
+      skip: true,
     }
   );
 
-  const errors = validate(entry) ?? {};
+  // Validate needs to make a init query, so we need to wait for it to be ready
+  const errors = !isLoading ? validate(entry) ?? {} : {};
```

Comment:
It's a code style thing but I think it might be more readable with something like:

```tsx
  if (isLoading) {
    return null;
  }

  const errors = validate(entry) ?? {};
  ```
or
```tsx
const errors = (!isLoading && validate(entry)) ?? {};
```